pkg/apis/componentconfig/types.go

```diff
@@ -712,3 +712,15 @@ type PersistentVolumeRecyclerConfiguration struct {
 	// in a multi-node cluster.
 	IncrementTimeoutHostPath int32 `json:"incrementTimeoutHostPath"`
 }
+
+const (
+	ApiServerConfigMapName string = "apiserver-config"
```

Comment:
Is it better for this to be const, or to make them flags to apiserver, so someone could actually test a new config by changing the `--config-map` flag?